tests/bsim/bluetooth/ll/bis/testcase.yaml

```diff
@@ -40,3 +40,16 @@ tests:
     harness: bsim
     harness_config:
       bsim_exe_name: tests_bsim_bluetooth_ll_bis_prj_vs_dp_conf
+  bluetooth.ll.bis_past:
+    extra_args: CONF_FILE=prj_past.conf
+    platform_allow:
+      - nrf52_bsim
+      - nrf5340bsim/nrf5340/
+      - nrf5340bsim/nrf5340/cpuapp
```

Comment:
If you want to build for the nrf5340bsim/nrf5340/cpuapp you'll need to use sysbuild (you can check other bsim tests building for it)